docs/mux-logging-service.md

```diff
@@ -41,30 +41,27 @@ There are environment variables for the logging-mux DC config:
   on each node.  This allows the per-node Fluentd collectors to read and ship
   logs off the node as fast as possible to the logging-mux service, which will
   normalize and enrich with Kubernetes metadata, and store in Elasticsearch.
-* `MUX_ALLOW_EXTERNAL` - `true`/`false` - default is `false` - if `true`, the
-  logging-mux service will be able to handle log records sent from outside of
-  the cluster, as described below.
+  This will also be able to process records sent from outside of the cluster,
+  if the service has exposed an externalIP.
 * `FORWARD_LISTEN_PORT` - default `24284` - port to listen for secure_forward
   protocol log messages
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
 These are environment variables for the logging-fluentd daemonset config:
 
-* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
-  will not send logs directly to Elasticsearch, instead it will use
-  `secure_forward` to send logs to mux.
-* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
-  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
-  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset
```

Comment:
@smarterclayton Do you want to be able to run fluentd in a mode where it ships the logs off of the node as fast as possible, doing minimal processing?  Is there ever a case where you would want to run fluentd in such a mode?  Note that we have not even begun to try to optimize the actual processing pipeline code in fluentd/mux, so we don't really know how well it will actually scale.  @portante is correct that currently, fluentd uses way, way too much cpu because the processing pipeline is incredibly poorly designed, so the only way we can scale the logging pipeline today is to push all of that processing onto each node.